portfolio/src/main/webapp/script.js

```diff
@@ -149,6 +149,13 @@ function createComment(comment) {
   dateElement.innerText = comment.date + " UTC";
   dateElement.className = 'span-comment span-comment-date';
 
+  // Create span element holding the user email who posted the comment.
+  const userEmailElement = document.createElement('a');
+  userEmailElement.innerText = comment.user.emailAddress;
+  userEmailElement.href = 'mailto:' + comment.user.emailAddress;
```

Comment:
optional: Especially now that you're allowing a click to open a new email with this user, I think it would be good to at least do a few checks to ensure the text entered into the auth box is at least structured like an email (e.g. has an @)

Reply:
In deployment, the dummy version of the login site is replaced with an actual login functionality through Google's login service, so this isn't as relevant in deployment. Since I don't know how to stop the user from registering an invalid email address on the dummy login page in localhost, I think I'll leave this implementation.